packages/@okta/vuepress-theme-prose/components/Home.vue

```diff
@@ -44,6 +15,25 @@
               </div>
             </div>
 
+            <div class="row">
+              <div class="col-12">
+                <h2 class="homepage--sub-heading homepage--color-main">
+                  Check out the event website for agenda, speakers + breakout
```

Comment:
This should be regular font (not bold)

I propose the text should be "Learn how to add security to your apps, APIs, and infrastructure. Register for free!"